.htaccess

```diff
@@ -1,8 +1,6 @@
 <IfModule mod_rewrite.c>
 RewriteEngine on
 
-RewriteCond %{HTTPS} !=on
```

Comment:
I used the gateway in my infrastructure. so we don't need any https to the service.